fixtures/jar-app/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/tmp-test-project-1380/openjdk8:testing
```

Comment:
We should use Maven resource filtering to replace tmp-test-project-1380 with current project.